tcf_website/views/search.py

```diff
@@ -0,0 +1,106 @@
+"""Views for search results"""
+import os
+import json
+import requests
+
+from django.shortcuts import render
+from django.contrib.auth.decorators import login_required
+
+@login_required
+def search(request):
+    """Search results view."""
+
+    # Set query
+    query = request.GET.get('q', '')
+
+    # Set endpoints
+    courses_search_endpoint = os.environ['ES_COURSE_SEARCH_ENDPOINT']
+    instructors_search_endpoint = os.environ['ES_INSTRUCTOR_SEARCH_ENDPOINT']
+
+    # Fetch results
+    response1 = fetch(query, courses_search_endpoint)
+    response2 = fetch(query, instructors_search_endpoint)
+
+    # Format results
+    courses = format_response(response1)
+    instructors = format_response(response2)
+
+    # Arguments for template
+    args = {
+        "courses" : courses,
+        "instructors" : instructors,
+        "query" : query
+    }
+
+    return render(request, 'search/search.html', args)
+
+
+def fetch(query, api_endpoint):
```

Comment:
maybe a more specific name? like `fetch_elasticsearch`? Up to you, though

Reply:
fixed, I agree... helps the reader understand better what it does